addons/account/models/account_move.py

```diff
@@ -3939,6 +3939,11 @@ def _get_tax_exigible_domain(self):
             ('tax_ids.tax_exigibility', '!=', 'on_payment'), # So: exigible if at least one tax from tax_ids isn't on_payment
         ]
 
+    def _get_protected_lock_date_fields(self):
+        """ Returns the names of the fields that should be protected by the accounting lock date
+        """
+        return ['account_id', 'journal_id', 'amount_currency', 'currency_id', 'partner_id']
```

Comment:
I would even go one step further to force thinking about which locks we need to take into account when inheriting

```python
def _get_lock_date_protected_fields(self):
        tax_fnames = ['debit', 'credit', 'tax_line_id', 'tax_ids', 'tax_tag_ids']
        fiscal_fnames = tax_fnames + ['account_id', 'journal_id', 'amount_currency', 'currency_id', 'partner_id']
    return {
        'tax': tax_fnames,
        'fiscal': fiscal_fnames,
    }

Reply:
@william-andre 
does the same not apply to `PROTECTED_FIELDS_RECONCILIATION` too?